Add pd-ctl  `balance-leader-scheduler` config command (#7961)
pingcap/docs: pd-control.md

>> scheduler resume balance-region-scheduler      // Continue to run the balance-region scheduler
>> scheduler resume all                           // Continue to run all schedulers
>> scheduler config balance-hot-region-scheduler  // Display the configuration of the balance-hot-region scheduler

> >> scheduler config balance-leader-scheduler set batch 3 // Set 3 as the size of the operator that the balance-leader scheduler can execute in batches

Reviewer: ```suggestion
>> scheduler config balance-leader-scheduler set batch 3 // Set the size of the operator that the balance-leader scheduler can execute in a batch to 3
```

> >> scheduler config balance-leader-scheduler set batch 3 // Set the size of the operators that the balance-leader scheduler can execute in batches to 3

Author: ```suggestion
>> scheduler config balance-leader-scheduler set batch 3 // Set 3 as the size of the operators that the balance-leader scheduler can execute in batches
```
